Tetrahedral_remeshing/include/CGAL/tetrahedral_remeshing.h

```diff
@@ -252,10 +253,46 @@ void tetrahedral_isotropic_remeshing(
                                  CGAL::parameters::all_default());
 }
 
+/*!
+* Converts the triangulation referenced in the input
+* `Mesh_complex_3_in_triangulation_3`.
+* @tparam Tr is the underlying triangulation for `Mesh_complex_3_in_triangulation_3`.
+*            It can be instantiated with any 3D regular triangulation of CGAL provided
+*            that its vertex and cell base classes are models of the concepts
+*            `MeshVertexBase_3` (refined by `RemeshingCellBase_3`)
+*            and `MeshCellBase_3` (refined by `RemeshingVertexBase_3`), respectively.
+* @tparam CornerIndex is the type of the indices for feature corners.
+*            If `c3t3` has been generated using `CGAL::make_mesh_3()`, it must match
+*            the `Corner_index` type of the model of the `MeshDomainWithFeatures_3` concept used for mesh generation.
+* @tparam CurveIndex is the type of the indices for feature curves.
+*            If `c3t3` has been generated using `CGAL::make_mesh_3()`, it must match
+*            the `Curve_index` type of the model of the `MeshDomainWithFeatures_3` concept used for mesh generation.
+* @tparam NamedParameters a sequence of \ref Remeshing_namedparameters "Named Parameters"
+*
+* @param c3t3 the complex containing the triangulation (of type `Tr`) to be remeshed.
+
+* @todo finish this doc
+*/
+template<typename Tr, typename CornerIndex, typename CurveIndex>
+typename Tr::Tr_Base
```

Comment:
@lrineau we have a pb here : `Tr_base` is not documented

Reply:
The triangulation type in the C3T3 is not really a template, it's a regular 3D triangulation of CGAL, so couldn't you just template the function with the `Vb` and `Cb` instead? Then you have:
```
Triangulation_3<Vb, Cb> convert(C3T3<Regular_triangulation_3<Cb, Vb>, ...)
```

Reply:
There is a third parameter: the lock_ds:
```
template < class GT, class Tds_, class Lock_data_structure_ >
class Triangulation_3
```

Reply:
That sounds good, thank you!